tests/subsys/storage/flash_map/src/main.c

```diff
@@ -26,13 +26,17 @@ void test_flash_area_get_sectors(void)
 	uint8_t wd[256];
 	uint8_t rd[256];
 	const struct device *flash_dev;
+	const struct device *flash_dev_a = FLASH_AREA_DEVICE(image_1);
 
 	rc = flash_area_open(FLASH_AREA_ID(image_1), &fa);
 	zassert_true(rc == 0, "flash_area_open() fail");
 
 	/* First erase the area so it's ready for use. */
 	flash_dev = flash_area_get_device(fa);
 
+	/* Device obtained by label should match the one from fa object */
```

Comment:
```suggestion
	/* Device obtained by label should match the one from flash_dev object */
```
Or is it supposed to be `fa` ?

Reply:
It is; at line 35 we extract the device, to flash_dev pointer, from fa that has been opened using the same label as used with FLASH_AREA_DEVICE.